lib/graphql/backend/fragment/user_fragment.graphql

```diff
@@ -0,0 +1,173 @@
+fragment BaseUserFragment on User {
+    _id
+    name
+    username
+    display_name
+    new_photos
+    image_avatar
+    wallets
+    wallet_custodial
+    blocked
+    blocked_expanded {
+        _id,
+        name,
+        image_avatar,
+    }
+}
+
+fragment UserProfileFragment on User {
+    created_at
+    display_name
+    first_name
```

Comment:
there are some duplicated fields  with BaseUserFragment, may check to prevent duplicate

Reply:
for file structure,
I suggest 
`lib/graphql/backend/domain/etc`
example for user domain then we can have `backend/user/fragment`
because there will be lots of domains, when multiple files are added will be hard to find